### PR TITLE
Document Silence layer in layer catalogue

### PR DESCRIPTION
**Pull request by katerakelly**
Add Silence layer documentation

### CODE REVIEW

File: docs/tutorial/layers.md [hunk 1]
```diff
@@ -486,3 +486,15 @@ The `SLICE` layer is a utility layer that slices an input layer to multiple outp
 #### Mean-Variance Normalization
 
 `MVN`
+
+#### Silence
+
+* LayerType: `SILENCE`
```
Comment (shelhamer) [inline docs/tutorial/layers.md]: The type is `Silence` in the current proto format.

File: docs/tutorial/layers.md [hunk 2]
```diff
@@ -486,3 +486,15 @@ The `SLICE` layer is a utility layer that slices an input layer to multiple outp
...
+* CPU implementation: `./src/caffe/layers/silence_layer.cpp`
+* CUDA GPU implementation: `./src/caffe/layers/silence_layer.cu`
+* Sample
+
+	layers { name: "silence" bottom: "input" type: SILENCE }
```
Comment (shelhamer) [inline docs/tutorial/layers.md]: Please update to the current proto format by `layers` -> `layer` and `type: SILENCE` -> `type: "Silence"`

**Comment by shelhamer** [comment]
Thanks for the documentation. Once the details fit the current proto format this can be merged.

**Comment by williford** [comment]
This information has been added in #5148. Thanks for the submission. There is still a lot of room for improvement for the layer documentation and would like to encourage future PRs for documentation!

**Pull request closed without merging**